### PR TITLE
tests(regression): accounting scenarios (burner)

### PR DESCRIPTION
**Pull request by madlabman**

### CODE REVIEW

File: tests/regression/test_accounting.py [hunk 1]
```diff
@@ -672,3 +782,18 @@ def _rebase_limit_wei(block_identifier: int) -> int:
...
     )
+
+
+def _shares_burn_limit_no_pooled_ether_changes(block_identifier: int | str = "latest") -> int:
+    """Get shares burn limit from oracle report sanity checker contract when no changes in pooled Ether are expected"""
+
+    rebase_limit = contracts.oracle_report_sanity_checker.getMaxPositiveTokenRebase(block_identifier=block_identifier)
+    rebase_limit_plus_1 = rebase_limit + LIMITER_PRECISION_BASE
+    total_pooled_ether = contracts.lido.getTotalPooledEther(block_identifier=block_identifier)
+    pooled_ether_rate = total_pooled_ether * LIMITER_PRECISION_BASE // total_pooled_ether
```
Comment (TheDZhon) [inline tests/regression/test_accounting.py:793]: this is a tautology for the corner case when `preTVL == postTVL`
can simplify to just:
```python
    return (
        contracts.lido.getTotalShares(block_identifier=block_identifier)
        * rebase_limit // rebase_limit_plus_1
    )
```

**Review by TheDZhon — COMMENTED** [review]
:eyes:
